taskcluster/ci/test/unit.yml

```diff
@@ -0,0 +1,39 @@
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+---
+task-defaults:
+    treeherder:
+        symbol: T(unit)
+        kind: test
+        tier: 1
+    worker:
+        max-run-time: 3600
+        chain-of-trust: true
+    run:
+        using: run-task
+        use-caches: true
+
+ctest-linux:
+    description: "Run full C-Test"
+    fetches:
+        toolchain:
+            - qt-linux
+    worker-type: b-linux

```

Comment:
With Firefox, we usually run tests on target OS' (usually this means Ubuntu for Linux). I see that these tests require doing a build first, so that's probably not viable here, but it may be worth considering for the future. (With Firefox, the build system is capable of packaging tests for consumption elsewhere - so the tests tasks end up downloading the build and the test packages.)